src/SoftekBarcodeAPI.php

```diff
@@ -0,0 +1,37 @@
+<?php
+declare(strict_types=1);
+
+namespace DexproSolutionsGmbh\SoftekBarcodeWrapper;
+
+use DexproSolutionsGmbh\SoftekBarcodeWrapper\Model\SoftekBarcodeConfig;
+use FFI;
+
+class SoftekBarcodeAPI
+{
+    /**
+     * The foreign C API
+     *
+     * @var FFI
+     */
+    protected FFI $interface;
+
+    /**
+     * The persistent instance of the barcode tool kit
+     *
+     * @var FFI\CData
+     */
+    protected FFI\CData $instance;
```

Comment:
I think `$interface` and `$instance` can lead to accidental usages of each other. Can we rename one of them so that we do not accidentally use the wrong field?

Reply:
I confirm with your comment, i think also that a more clear declaration of the class properties can prevent an accidental use. Thanks !